app-react/src/main/kotlin/com/ustadmobile/view/CourseTerminologyEditScreen.kt

```diff
@@ -0,0 +1,104 @@
+package com.ustadmobile.view
+
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.hooks.useStringsXml
+import com.ustadmobile.core.impl.locale.StringsXml
+import com.ustadmobile.core.viewmodel.CourseTerminologyEditUiState
+import com.ustadmobile.lib.db.entities.TerminologyEntry
+import com.ustadmobile.mui.components.UstadTextEditField
+import mui.material.List
+import mui.material.ListItem
+import mui.system.Container
+import mui.system.Stack
+import mui.system.StackDirection
+import mui.system.responsive
+import react.FC
+import react.Props
+import react.useState
+
+external interface CourseTerminologyEditScreenProps : Props {
+
+    var uiState: CourseTerminologyEditUiState
+
+    var onTerminologyTermChanged: (TerminologyEntry?) -> Unit
+
+    var onCtTitleChanged: (String?) -> Unit
+
+}
+
+val CourseTerminologyEditScreenComponent2 = FC <CourseTerminologyEditScreenProps> { props ->
+
+    val strings: StringsXml = useStringsXml()
+
+    Container {
+        maxWidth = "lg"
+
+        Stack {
+            direction = responsive(StackDirection.column)
+            spacing = responsive(2)
+
+            UstadTextEditField {
+                value = props.uiState.entity?.ctTitle ?: ""
+                label = strings[MessageID.name]
+                error = props.uiState.titleError
+                enabled = props.uiState.fieldsEnabled
+                onChange = {
+                    props.onCtTitleChanged(it)
+                }
+            }
+
+            + strings[MessageID.your_words_for]
+
+            List {
```

Comment:
This should not use List/ListItem as above - we don't have a List with header and subheaders as per the materialdesign spec. We just have a column / stack of textfields.